symengine/codegen.h

```diff
@@ -155,7 +155,7 @@ class CodePrinter : public BaseVisitor<CodePrinter, StrPrinter>
     void bvisit(const Constant &x)
     {
         if (eq(x, *E)) {
-            str_ = "M_E";
+            str_ = "exp(1)";
```

Comment:
Can you add a test for this?

Reply:
Done. Should I change it for M_PI as well?

Reply:
You can do it in another PR